### DISCUSSION TITLE
Reverse proxy with auth.

### DISCUSSION THREAD

**Discussion by derekschrock** [iamcco/markdown-preview.nvim#556] [Q&A]
From the question asked in #210 by @ysl2 for reverse proxying.

Take your pick.

https://httpd.apache.org/docs/2.4/howto/reverse_proxy.html
https://httpd.apache.org/docs/2.4/howto/auth.html

https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/
https://docs.nginx.com/nginx/admin-guide/security-controls/configuring-http-basic-authentication/

https://docs.haproxy.org/2.7/intro.html#3.3.1
https://www.haproxy.com/documentation/hapee/latest/security/authentication/basic-authentication/

**Comment by ysl2** [comment]
Thanks bro !
